pkg/migration/controllers/migration.go

```diff
@@ -2131,30 +2137,28 @@ func (m *MigrationController) getMigrationSummary(migration *stork_api.Migration
 		migrationSummary.ElapsedTimeForVolumeMigration = elapsedTimeVolume
 	}
 
-	if migration.Spec.IncludeResources == nil || *migration.Spec.IncludeResources {
```

Comment:
We should calculate the MigrationSummary and populate the time taken for migration k8s resources,  even if IncludeResources is set to false, since we would be migrating the PV and PVC objects.